package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "edx-pattern-library",
-  "version": "0.18.1",
+  "version": "0.18.2-stanford.3",
```

Comment:
Why `.3`? Isn't this our first (stanford) release of `0.18.2`?

Reply:
because we already have other tags for `0.18.2` that don't have the requisite code... I was silly and created tags too early...